packages/backend/src/jwt/__tests__/verifyJwt.test.ts

```diff
@@ -12,123 +10,120 @@ import {
   signedJwt,
   someOtherPublicKey,
 } from '../../fixtures';
-import { assertOk } from '../../util/testUtils';
 import { decodeJwt, hasValidSignature, verifyJwt } from '../verifyJwt';
 
-export default (QUnit: QUnit) => {
-  const { module, test } = QUnit;
-  const invalidTokenError = {
-    reason: 'token-invalid',
-    message: 'Invalid JWT form. A JWT consists of three parts separated by dots.',
-  };
-
-  module('hasValidSignature(jwt, key)', () => {
-    test('verifies the signature with a JWK formatted key', async assert => {
-      const { data: decodedResult } = decodeJwt(signedJwt);
-      assertOk<Jwt>(assert, decodedResult);
-      const { data: signatureResult } = await hasValidSignature(decodedResult, publicJwks);
-      assert.true(signatureResult);
-    });
-
-    test('verifies the signature with a PEM formatted key', async assert => {
-      const { data: decodedResult } = decodeJwt(signedJwt);
-      assertOk<Jwt>(assert, decodedResult);
-      const { data: signatureResult } = await hasValidSignature(decodedResult, pemEncodedPublicKey);
-      assert.true(signatureResult);
-    });
-
-    test('it returns false if the key is not correct', async assert => {
-      const { data: decodedResult } = decodeJwt(signedJwt);
-      assertOk<Jwt>(assert, decodedResult);
-      const { data: signatureResult } = await hasValidSignature(decodedResult, someOtherPublicKey);
-      assert.false(signatureResult);
-    });
+const invalidTokenError = {
+  reason: 'token-invalid',
+  message: 'Invalid JWT form. A JWT consists of three parts separated by dots.',
+};
+
+describe('hasValidSignature(jwt, key)', () => {
+  it('verifies the signature with a JWK formatted key', async () => {
+    const { data: decodedResult } = decodeJwt(signedJwt);
+    // @ts-expect-error
+    const { data: signatureResult } = await hasValidSignature(decodedResult, publicJwks);
```

Comment:
👀 curious why we need these `@ts-expect-error`s now where we didn't previously

Reply:
Fixed